DesignSafe-CI/portal
# task/DES-1954: Update favicon

## Overview: ##
The current tab icon for DesignSafe is blurry, Matt S has provided a new icon that is much simpler and should be recognizable in a large number of open tabs.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1954](https://jira.tacc.utexas.edu/browse/DES-1954)

## Testing Steps: ##
1. Load the portal locally
2. Run `./manage.py collectstatic -i demo` in `des_django`
3. Confirm the new favicon is visible in the browser tab

## UI Photos:
<img width="248" alt="Screen Shot 2021-05-19 at 10 13 26 AM" src="https://user-images.githubusercontent.com/20326896/118838215-1e1b4280-b88b-11eb-8308-1cb1036eba8a.png">
